packages/hardhat-core/test/internal/hardhat-network/provider/modules/evm.ts

```diff
@@ -891,7 +891,7 @@ describe("Evm module", function () {
             });
 
             it("should mine block with transaction after the interval", async function () {
-              const interval = 1000;
+              const interval = 100;
```

Comment:
This is unrelated, but since I was optimizing the time of this file, I just did it. I think the test is equally valid with a 100ms interval, and this reduces the total test time by another 1800ms.